fixtures/workers-with-assets-and-service-bindings/workerWS/src/index.ts

```diff
@@ -0,0 +1,20 @@
+import { WorkerEntrypoint } from "cloudflare:workers";
+
+export default class extends WorkerEntrypoint {
+	fetch(req: Request) {
+		if (!req.headers.get("Upgrade")?.includes("websocket")) {
+			return new Response("Hello from worker-ws fetch()");
+		}
+
+		const [server, client] = Object.values(new WebSocketPair());
+
+		server.accept();
+
+		const textEncoder = new TextEncoder();
+		server.addEventListener("message", (event) => {
+			server.send(textEncoder.encode(`pong: ${event.data}`));
```

Comment:
should we then call `server.close()` here?

Reply:
I wondered about that as well. I'm basically following the steps in https://developers.cloudflare.com/workers/examples/websockets/, which does not mention closing the server. Perhaps in our case it makes more sense since we know the client won't send any other messages after the initial one. Happy to add it

Reply:
added in https://github.com/cloudflare/workers-sdk/pull/8612/commits/c2a8365a88aaa16b8861957bc6d7cd0952cf0e70